csrc/tma.cpp

```diff
@@ -104,6 +104,12 @@ inline CUtensorMapFloatOOBfill getCUtensorMapFloatOOBfill(
 
 using TensorMap = CUtensorMap;
 
+#else
+
+struct TensorMap {
```

Comment:
Is TMA supported with CUDA 11? Or is this to just make the build pass?

Reply:
This is just to make the build pass.

Reply:
OK, please leave a comment

Reply:
added